benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -567,6 +567,25 @@ int main(int argc, char *argv[])
       throw std::runtime_error("HIP backend not available!");
 #endif
 
+#ifdef KOKKOS_ENABLE_OPENMPTARGET
+    if (spec.backends == "all" || spec.backends == "openmptarget")
+      register_benchmark<
+          ArborX::BVH<Kokkos::Experimental::OpenMPTarget::device_type>>(
+          "ArborX::BVH<OpenMPTarget>", spec);
+#else
+    if (spec.backends == "openmptarget")
+      throw std::runtime_error("OpenMPTarget backend not available!");
+#endif
+
+#ifdef KOKKOS_ENABLE_
```

Comment:
Oops

Reply:
Noticed that today, but thought it was intentional to enable SYCL by default.